pset0/wc61.cc

```diff
@@ -0,0 +1,40 @@
+#include <cstdint>
+#include <cstdio>
+#include <cctype>
+
+
+int main() {
+  uint32_t bytes = 0;
+  uint32_t words = 0;
+  uint32_t lines = 0;
+
+  bool last_was_space = true;
+  bool curr_is_space = false;
+  bool last_was_newline = false;
+
+  char c;
+  while (true) {
+    c = fgetc(stdin);
+    if (c == EOF) {
+      if (!last_was_newline && bytes > 0) {
```

Comment:
This is fine if you wan't to count a sequence of words not ending with '\n' as a line. It won't match wc output, but it's okay. The problem statement though defines a line as ending with '\n'.

Reply:
I should have properly read the spec in the README.
I have now modified the program to behave like wc.